### PR TITLE
machines: new "Disks" subtab

### PR DESCRIPTION
**Pull request by mareklibra**
VM detail is enriched for new "Disks" subtab listing all disks assigned to particular VM.

### CODE REVIEW

**Comment by mareklibra** [comment]
disks

File: pkg/lib/listing.less [hunk 1]
```diff
@@ -33,55 +33,55 @@ table.listing-ct > thead td > a {
     padding: @listing-ct-padding 0 @listing-ct-padding;
 }
 
-table.listing-ct thead th {
+table.listing-ct > thead th {
```
Comment (stefwalter) [inline pkg/lib/listing.less]: Have you validated that the rest of Cockpit behaves well with these changes? @garrett Could you give it a look over too?

**Comment by stefwalter** [comment]
Looks good other than that :)

File: pkg/lib/listing.less (hunk 1 above)
Comment (mareklibra) [inline pkg/lib/listing.less]: I don't see any difference

Comment (mareklibra) [inline pkg/lib/listing.less]: Checked in Vagrant - other usage of the listing component seem to be untouched (i.e. on the "React Patterns" page)

**Comment by mareklibra** [comment]
Integration test for the new "Disks" subtab added.

**Comment by mareklibra** [comment]
Failing tests are unrelated

**Comment by dperpeet** [comment]
I'm working on a follow-up

**Comment by dperpeet** [comment]
replaced by #6067

**Pull request closed without merging**